app/controllers/houston/ci/test_runs_controller.rb

```diff
@@ -52,6 +52,15 @@ def save_results
         head :ok
       end
 
+      def save_results_async
+        with_results_url do |results_url|
+          Rails.logger.debug "Scheduling pulling from #{results_url} in 5s"
```

Comment:
This log statement doesn't match `wait: 10.seconds`.

Reply:
🤦 Oops. Yeah, I changed that at the last minute, since 5s proved to not be long enough to wait. I'll update the log statement.